### PR TITLE
Additional resource to Rails Basics - Views lesson

### PR DESCRIPTION
**Pull request by egemen-github**
Added a YouTube video link from freeCodeCamp channel to additional resources section. This video's first 60 minute covers what curriculum teaches up until this lesson and it is a very good wrap up for Rails basics.

Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`

Complete the following checkboxes ONLY IF they are applicable to your PR. You can complete them later if they are not currently applicable:
-   [x] I have previewed all lesson files included in this PR with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure the Markdown content is formatted correctly
-   [x] I have ensured all lesson files included in this PR follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

<hr>

**1. Because:**

Closes #XXXXX

This PR is for adding an additional video to _Rails Basics - Views_ lesson. I personally benefitted this video (video from freeCodeCamp) when I was that at the Rails Basics - Views lesson. 

I wanted to add this video link to Views lesson, because this video is almost perfectly aligned with our curriculum's topic order. Especially first 60 minutes of the video is wrapping up what has been taught about Rails Basics e.g., `routing`, `MVC`, `CRUD`, `partials` and `link_to`. 

I also indicated that first 60 minutes is recommended, if someone wants to watch until end, it's okay too. I want others to use utilize this video.

**2. This PR:**

* A video link added to additional resources section in Rails Basics - Views lesson.

**3. Additional Information:**

### CODE REVIEW

**Comment by ChargrilledChook** [comment]
Hey @egemen-github, thanks for this submission.
If you got good value out of this link I've got no problems adding it in, but I think it needs a little bit more context given how long it is.

Perhaps something like "If you'd like a deeper dive into <relevant concepts> watch the first 60 minutes of this video which covers A B C"

What do you think?

**Comment by egemen-github** [comment]
@ChargrilledChook Definitely. I watched this video and really liked it. It's 4 hours long but no need to watch more than 60 mins in my opinion.

> Perhaps something like "If you'd like a deeper dive into <relevant concepts> watch the first 60 minutes of this video which covers A B C"

_Maybe like this_: If you'd like a deeper dive into `routing, MVC, CRUD and partials` watch the first 60 minutes of this video which provides a good warp up for what we've been learning so far, you'll get to create a simple Rails app from scratch."

Would this be ok?

**Comment by ChargrilledChook** [comment]
@egemen-github Sounds great 👍

**Comment by egemen-github** [comment]
@ChargrilledChook Awesome! So, I have a question. What do need to do now?

File: ruby_on_rails/rails_basics/views.md [hunk 1]
```diff
@@ -230,6 +230,7 @@ This section contains helpful links to other content. It isn't required, so cons
 * [Stack Overflow Post on Views](https://stackoverflow.com/questions/14429910/an-alternate-explanation-to-rails-layouts-rendering-partials-templates-and-v)
 * [Video on the Relationship Between Views and Controllers](https://www.youtube.com/watch?v=mRJSovhdzWc&ab_channel=GoRails)
 * [Video on ERB Tags](https://www.youtube.com/watch?v=na28woOGPUw&ab_channel=NoobandTube) - (this video will require you to turn your volume up)
+* [Introduction to Ruby on Rails from freeCodeCamp](https://youtu.be/fmyvWz5TUWg) - (first 60 minutes is recommended)
```
Comment (ChargrilledChook) [inline ruby_on_rails/rails_basics/views.md:233]: Pop your changes here :smiley:

**Review by ChargrilledChook — CHANGES_REQUESTED** [review]
Add the changes we discussed and this will be good to go :tada:

**Comment by ChargrilledChook** [comment]
@egemen-github Good question! I normally edit these from a branch, not directly from the repo so I had to have a bit of a poke around. I believe you can edit this "branch" directly from github by going to the file tab, clicking the `...` on the right and going to edit file ie

image

Give that a try and feel free to ping me when it's done or if you need any more guidance

**Comment by egemen-github** [comment]
Thank you for helping me out :) I've made the change. How does it look?

File: ruby_on_rails/rails_basics/views.md [hunk 2]
```diff
@@ -230,7 +230,7 @@ This section contains helpful links to other content. It isn't required, so cons
 * [Stack Overflow Post on Views](https://stackoverflow.com/questions/14429910/an-alternate-explanation-to-rails-layouts-rendering-partials-templates-and-v)
 * [Video on the Relationship Between Views and Controllers](https://www.youtube.com/watch?v=mRJSovhdzWc&ab_channel=GoRails)
 * [Video on ERB Tags](https://www.youtube.com/watch?v=na28woOGPUw&ab_channel=NoobandTube) - (this video will require you to turn your volume up)
-* [Introduction to Ruby on Rails from freeCodeCamp](https://youtu.be/fmyvWz5TUWg) - (first 60 minutes is recommended)
+* [Introduction to Ruby on Rails from freeCodeCamp](https://youtu.be/fmyvWz5TUWg) - If you'd like a deeper dive into routing, MVC, CRUD and partials watch the first 60 minutes of this video which provides a good warp up for what we've been learning so far, you'll get to create a simple Rails app from scratch.
```
Comment (egemen-github) [inline ruby_on_rails/rails_basics/views.md:233]: @ChargrilledChook it's right here.

Comment (ChargrilledChook) [reply ruby_on_rails/rails_basics/views.md:233]: Thanks @egemen-github , looks good. Two minor things - minor typo (warp -> wrap) and I think we should break the sentence up a little bit. What do you think?

```suggestion
* [Introduction to Ruby on Rails from freeCodeCamp](https://youtu.be/fmyvWz5TUWg) - If you'd like a deeper dive into routing, MVC, CRUD and partials watch the first 60 minutes of this video which provides a good wrap up for what we've been learning so far. You'll also get to create a simple Rails app from scratch if you'd like some extra practice.
```

Comment (egemen-github) [reply ruby_on_rails/rails_basics/views.md:233]: I agree. I think it is better with the suggested changes.

**Comment by egemen-github** [comment]
You can check the last change. I hope it's all good now.

**Comment by ChargrilledChook** [comment]
@egemen-github Merged, thanks for the contribution 🚀

**Comment by egemen-github** [comment]
Thanks

**Pull request merged**